build.gradle

```diff
@@ -30,8 +30,8 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 ext {
-    cradleVersion       = '3.2.2'
-    junitVersion      = '5.8.2'
+    cradleVersion       = '3.1.3-TH2-3827-2909433977-SNAPSHOT'
```

Comment:
Why did we downgrade the version?

Reply:
I used this version according to task - "Update cradle version to the one from [TH2-3827]". Looks like there is no newer version which contains eventBatchDurationMillis parameter.